src/contracts/LidoARM.sol

```diff
@@ -38,7 +38,7 @@ contract LidoARM is Initializable, AbstractARM {
     /// @param _lidoWithdrawalQueue The address of the Lido's withdrawal queue contract
     /// @param _claimDelay The delay in seconds before a user can claim a redeem from the request
     constructor(address _steth, address _weth, address _lidoWithdrawalQueue, uint256 _claimDelay)
-        AbstractARM(_weth, _steth, _weth, _claimDelay)
+        AbstractARM(_weth, _steth, _weth, _claimDelay, 0)
```

Comment:
Should we have a non-zero value here since this is going to be hard-coded into the next deploy?

Reply:
The `minSharesToRedeem` was introduced to get around an issue with the Silo vaults which failed if a small amount of shares was redeemed. eg 100 wei of shares

We'll have to see if the lending market used by LidoARM will have the same problem.